tiltIndicator_tiltIndicatorAfter_workflow.Rmd

```diff
@@ -0,0 +1,213 @@
+---
+title: "tiltInidicator_tiltIndicatorAfter_workflow"
+author: "Kalash"
+date: "2023-07-21"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+The tiltIndicatorAfter_runner script uses the output from the tiltIndicatorBefore package as the inputs to the tiltIndicator and tiltIndicatorAfter packages. This script uses the CSV files from the tiltIndicatorBefore package to create the final output of the four indicators. 
+
+## Setup
+```{r}
+library(dplyr, warn.conflicts = FALSE)
+library(readr, warn.conflicts = FALSE)
+library(tidyr, warn.conflicts = FALSE)
+devtools::install_github("2DegreesInvesting/tiltIndicator")
+library(tiltIndicator)
+devtools::install_github("2DegreesInvesting/tiltIndicatorAfter")
+library(tiltIndicatorAfter)
+```
+
+## tiltIndicatorAfter data common for all indicators
+
+`ep_companies`, `ei_activities_overview`, `mapper_ep_ei`, `ei_input_data`, and `isic_4digit_name` are the five datasets which are used as the input for the tiltIndicatorAfter package. They all are the output from tiltIndicatorBefore package. These five datasets are used for adding additional data to all the four indicators (Emissions profile, Emissions profile upstream, Sector profile, Sector profile upstream). 
```

Comment:
- [ ] While this is true, it is also brittle. The names of those datasets will likely change. Best to refer to those names as little as possible so that this file won't become outdated so easily.